cmd/crane/cmd/util.go

```diff
@@ -21,6 +21,35 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 )
 
+type platformsValue struct {
+	platforms []v1.Platform
+}
+
+func (ps *platformsValue) Set(platform string) error {
+	if ps.platforms == nil {
+		ps.platforms = []v1.Platform{}
+	}
+	p, err := parsePlatform(platform)
```

Comment:
Should we use `v1.ParsePlatform` instead?

Reply:
Nice.